lib/ansible/modules/cloud/vmware/vsphere_copy.py

```diff
@@ -7,17 +7,19 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-ANSIBLE_METADATA = {'metadata_version': '1.1',
-                    'status': ['preview'],
-                    'supported_by': 'community'}
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
 
 
 DOCUMENTATION = '''
 ---
 module: vsphere_copy
 short_description: Copy a file to a VMware datastore
 description:
-    - Upload files to a VMware datastore through a VCenter of a ESXi
+    - Upload files to a VMware datastore through a VCenter REST API.
```

Comment:
I would like to make clear that this module works perfectly fine with a single ESXi. I wrote, 'of', instead of 'or' :man_facepalming:.